src/WPF/ArcGISRuntime.WPF.Samples/Samples/LocalServer/LocalServerServices/LocalServerServices.xaml.cs

```diff
@@ -38,8 +38,18 @@ public LocalServerServices()
 
         private void Initialize()
         {
-            // Subscribe to event notification for the local server instance
-            LocalServer.Instance.StatusChanged += (o, e) => { UpdateUiWithServiceUpdate("Local Server", e.Status); };
+            try
+            {
+                // Subscribe to event notification for the local server instance
+                LocalServer.Instance.StatusChanged += (o, e) =>
+                {
+                    UpdateUiWithServiceUpdate("Local Server", e.Status);
+                };
+            }
+            catch (Exception ex)
+            {
+                MessageBox.Show(String.Format("Please ensure that Local Server is installed prior to loading this sample. See the readme or metadata.json for more info. Message: {0}", ex.Message), "Local Server failed to start");
```

Comment:
@nCastle1 I recommend making the error message match the message strings above (`Please ensure that local server is installed prior to using the sample. See instructions in readme.me or metadata.json.`)